src/libfetchers/git-utils.cc

```diff
@@ -206,7 +206,8 @@ static void initRepoAtomically(std::filesystem::path & path, bool bare)
     if (pathExists(path.string()))
         return;
 
-    Path tmpDir = createTempDir(os_string_to_string(PathViewNG{std::filesystem::path(path).parent_path()}));
+    std::filesystem::path tmpDir =
+        createTempDir(os_string_to_string(PathViewNG{std::filesystem::path(path).parent_path()}));
```

Comment:
```suggestion
        createTempDir(path.parent_path());
```
Can that just be this now?